central/notifiers/email/email.go

```diff
@@ -592,9 +592,9 @@ func PlainTextAlert(alert *storage.Alert, uiEndpoint string, mitreStore mitreDS.
 	return notifiers.FormatAlert(alert, alertLink, funcMap, mitreStore)
 }
 
-func BuildReportMessage(recipients []string, from, subject, messageText string, zippedReportData *bytes.Buffer) Message {
+func BuildReportMessage(recipients []string, from, subject, messageText string, zippedReportData *bytes.Buffer, reportName string) Message {
 	if subject == "" {
-		subject = fmt.Sprintf("%s Image Vulnerability Report for %s", branding.GetProductNameShort(), time.Now().Format("02-January-2006"))
+		subject = fmt.Sprintf("%s %s for %s", branding.GetProductNameShort(), reportName, time.Now().Format("02-January-2006"))
```

Comment:
Should we truncate the reportName if it is too long? If it is longer than say 80 characters , then we set `reportName = reportName[0:80] + "..."`.

Reply:
Doing this, and the suggestion below, and adding a unit test for that function